aws-api-gateway/dashboards/Page_AWS CloudFront.json

```diff
@@ -0,0 +1,2074 @@
+[ {
```

Comment:
incorrect dashboard but we don't need these here anyway, you can delete the dashboards sub directory

Reply:
Yes